jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/ControlPanel.tsx

```diff
@@ -0,0 +1,112 @@
+import { Grid, Paper } from '@material-ui/core';
```

Comment:
This is the control panel (left panel) for the curation view. It displays the labels for the currently selected units and allows adding/removing of labels using single mouse clicks.